README.md

```diff
@@ -62,6 +62,30 @@ python -m expertise.tfidf_scores config.json
 
 The output will generate a `.csv` file with the name pattern `<config_name>-scores.csv`.
 
+## Detect Duplicates
```

Comment:
I know that we support duplicate detection for both within a venue and between different venues. I can see that we basically compare the data in the directories specified by `config.json` params `submissions_path` and `other_submissions_path`.
I think it would be a good idea to add a line or two here explaining how these params can be used to support both duplicate detection modes.

Reply:
I am not really sure what you mean by explaining the two modes. I can avoid having to make the user duplicate the submissions into other_submissions folder when it is intraconference duplicate detection, but I am not sure what you would like me to explain in the README.

Reply:
By 2 modes I meant intra-conference and inter-conference.

Reply:
Got it! I have updated the README, let me know if you think that the explanation is clear enough.